package.json

```diff
@@ -78,11 +78,11 @@
     "xmlbuilder": "^2.6.1"
   },
   "devDependencies": {
-    "browserify": "^9.0.3",
+    "browserify": "^10.1.3",
```

Comment:
@gero3 any way for you to test that this PR (after generating the bundle) works with your mr doob app?